trace-forward/src/Trace/Forward/Configuration.hs

```diff
@@ -0,0 +1,55 @@
+module Trace.Forward.Configuration
+  ( AcceptorConfiguration (..)
+  , ForwarderConfiguration (..)
+  , HowToConnect (..)
+  , Host
+  , Port
+  ) where
+
+import           Control.Tracer (Tracer)
+import           Data.IORef (IORef)
+import           Data.Text (Text)
+import           Data.Word (Word16)
+import           Ouroboros.Network.Driver (TraceSendRecv)
+
+import           Trace.Forward.Protocol.Type (NodeInfo, Request, TraceForward)
+
+type Host = Text
+type Port = Word16
+
+-- | Specifies how to connect to the peer.
+data HowToConnect
+  = LocalPipe    !FilePath    -- ^ Local pipe (UNIX or Windows).
+  | RemoteSocket !Host !Port  -- ^ Remote socket (host and port).
+
+-- | Acceptor configuration, parametrized by trace item's type.
+data AcceptorConfiguration lo = AcceptorConfiguration
+  { -- | The tracer that will be used by the acceptor in its network layer.
+    acceptorTracer    :: !(Tracer IO (TraceSendRecv (TraceForward lo)))
+    -- | The endpoint that will be used to listen to the forwarder.
+  , forwarderEndpoint :: !HowToConnect
+    -- | The request specifies how many 'TraceObject's will be requested.
+  , whatToRequest     :: !Request
+    -- | Additional action that will be performed every time the acceptor will
+    -- receive the reply from the forwarder.
```

Comment:
I think this should be "Additional action that will be performed every time the acceptor receives a reply from the forwarder"